pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "m2c"
+version = "0.1.0"
+description = "MIPS and PowerPC decompiler"
+authors = [
+    "Matt Kempster <kidpixel@gmail.com>",
+    "Simon Lindholm <simon.lindholm10@gmail.com>",
+    "Zach Banks <zjbanks@gmail.com>",
+    "Ethan Roseman <ethteck@gmail.com>"
+]
+license = "GPL-3.0-only"
+readme = "README.md"
+repository = "https://github.com/matt-kempster/m2c"
+packages = [{include = "m2c"}]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+pycparser = "^2.21"
+graphviz = {version = "^0.20.1", optional = true}
```

Comment:
apparently `optional=true` dependencies are useless unless you have an "extras" section with an extra -> dep mapping that allows users to do `poetry install -E theextra`. So I guess either add one of those or make the dep non-optional.